From 8113fec0e3d5cdca1756386c1cee56bdfb47f641 Mon Sep 17 00:00:00 2001
From: Ben Greear <greearb@candelatech.com>
Date: Fri, 21 May 2021 15:57:40 -0700
Subject: [PATCH 08/15] intel-oot:  comment out a few too-new things (fixme)

These should be re-enabled when we move to new kernel
(or possibly backport more features into this one)

Signed-off-by: Ben Greear <greearb@candelatech.com>
---
 drivers/net/wireless/intel-oot/iwlwifi/mvm/ftm-initiator.c | 5 +++--
 drivers/net/wireless/intel-oot/iwlwifi/mvm/mac-ctxt.c      | 7 ++++---
 2 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/drivers/net/wireless/intel-oot/iwlwifi/mvm/ftm-initiator.c b/drivers/net/wireless/intel-oot/iwlwifi/mvm/ftm-initiator.c
index fe0a60f6dfc0b..c214cff6dfca5 100644
--- a/drivers/net/wireless/intel-oot/iwlwifi/mvm/ftm-initiator.c
+++ b/drivers/net/wireless/intel-oot/iwlwifi/mvm/ftm-initiator.c
@@ -895,8 +895,9 @@ static int iwl_mvm_ftm_start_v13(struct iwl_mvm *mvm,
		if (err)
			return err;

-		if (peer->ftm.trigger_based || peer->ftm.non_trigger_based)
-			target->bss_color = peer->ftm.bss_color;
+		// TODO:  Re-enable this. --Ben
+		//if (peer->ftm.trigger_based || peer->ftm.non_trigger_based)
+		//	target->bss_color = peer->ftm.bss_color;

		if (peer->ftm.non_trigger_based) {
			target->min_time_between_msr =
diff --git a/drivers/net/wireless/intel-oot/iwlwifi/mvm/mac-ctxt.c b/drivers/net/wireless/intel-oot/iwlwifi/mvm/mac-ctxt.c
index c636ce3e63616..3e58873517e0b 100644
--- a/drivers/net/wireless/intel-oot/iwlwifi/mvm/mac-ctxt.c
+++ b/drivers/net/wireless/intel-oot/iwlwifi/mvm/mac-ctxt.c
@@ -661,9 +661,10 @@ static int iwl_mvm_mac_ctxt_cmd_sta(struct iwl_mvm *mvm,
		if (vif->bss_conf.twt_protected)
			ctxt_sta->data_policy |=
				cpu_to_le32(PROTECTED_TWT_SUPPORTED);
-		if (vif->bss_conf.twt_broadcast)
-			ctxt_sta->data_policy |=
-				cpu_to_le32(BROADCAST_TWT_SUPPORTED);
+		// TODO:  Re-enable this. --Ben
+		//if (vif->bss_conf.twt_broadcast)
+		//	ctxt_sta->data_policy |=
+		//		cpu_to_le32(BROADCAST_TWT_SUPPORTED);
	}


--
2.26.3
